src/mcpdotnet/Client/IMcpClient.cs

```diff
@@ -8,11 +8,6 @@ namespace McpDotNet.Client;
 /// </summary>
 public interface IMcpClient : IAsyncDisposable
 {
-    /// <summary>
-    /// Gets a value indicating whether the client has been initialized.
-    /// </summary>
-    bool IsInitialized { get; }
```

Comment:
Does this mean that initialization is the responsibility of the factory? Was this prompted by MEAI or an opportunistic fix?

Reply:
> Does this mean that initialization is the responsibility of the factory?

Yes, it already was as of a previous PR that moved connecting into CreateAsync. This was a leftover I just noticed.